app/src/main/res/layout/fragment_login.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
We usually the `xmlns:android="http://schemas.android.com/apk/res/android"` in the next line. Is a "wolox" convention. If you run a format code, it will fix it the way you push your code, so remember to check this things. It is something like this:
```
<androidx.constraintlayout.widget.ConstraintLayout 
    xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:tools="http://schemas.android.com/tools"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    android:gravity="center"
    android:orientation="vertical"
```

As you can see, is just to add a newLine